osbs/api.py

```diff
@@ -351,25 +352,30 @@ def create_prod_build(self, git_uri, git_ref,
         :param git_ref: str, reference to commit
         :param git_branch: str, branch name (may be None)
         :param user: str, user name
-        :param component: str, component name
-        :param target: str, koji target (may be None)
+        :param component: str, not used anymore
+        :param target: str, koji target
         :param architecture: str, build architecture
         :param yum_repourls: list, URLs for yum repos
         :param koji_task_id: int, koji task ID requesting build
         :param scratch: bool, this is a scratch build
         :return: BuildResponse instance
         """
+
+        def get_required_label(labels, names):
```

Comment:
Maybe add this?:
```
assert names  # always called with a non-empty literal tuple so names[-1] is safe
```
Then when reviewing this later when we've all forgotten what it's about, we'll know that `names[-1]` is safe. This sort of pattern (using a variable defined in a for loop) is the kind of thing a static analysis tool will wonder about too.